assets/js/react/components/common/LoginForm/index.jsx

```diff
@@ -0,0 +1,146 @@
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import styled from 'styled-components';
+
+import Translate from '../Translate';
+
+const FormWrapper = styled.form`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 0;
+`;
+
+const LoginButton = styled(Button)`
+  display: block;
+  margin: auto;
+`;
+
+const CircularProgressCentered = styled(CircularProgress)`
+  display: block;
+  margin: auto;
+`;
+
+const ErrorText = styled(Typography)`
+  ${({ theme }) => `
+  background-color: ${theme.palette.errorColor};
+  border-radius: 3px;
+  color: ${theme.palette.common.white};
+  padding: 0.5rem;
+  `}
+`;
+
+const LoginForm = ({
+  email,
+  onEmailChange,
+  password,
+  onPasswordChange,
+  onLogin,
+  isFetching,
+  authError,
+}) => {
+  const [isPasswordVisible, setIsPasswordVisible] = React.useState(false);
+
+  const toggleIsPasswordVisible = () => {
+    setIsPasswordVisible(!isPasswordVisible);
+  };
+  const handleMouseDownPassword = (event) => {
+    event.preventDefault();
+  };
+
+  const handleEmailChange = (event) => {
+    onEmailChange(event.target.value);
+  };
+  const handlePasswordChange = (event) => {
+    onPasswordChange(event.target.value);
+  };
+
+  return (
+    <>
+      <FormWrapper onSubmit={onLogin}>
+        <FormControl variant="filled">
+          <InputLabel htmlFor="input-with-icon-adornment">
+            <FormattedMessage id="Email" />
+          </InputLabel>
+          <FilledInput
+            name="email"
+            value={email}
+            onChange={handleEmailChange}
+            required
+          />
+        </FormControl>
+
+        <FormControl variant="filled">
+          <InputLabel htmlFor="filled-adornment-password">
+            <FormattedMessage id="Password" />
+          </InputLabel>
+          <FilledInput
+            name="password"
+            type={isPasswordVisible ? 'text' : 'password'}
+            value={password}
+            onChange={handlePasswordChange}
+            endAdornment={
+              <InputAdornment position="end">
+                <IconButton
+                  aria-label="toggle password visibility"
+                  onClick={toggleIsPasswordVisible}
+                  onMouseDown={handleMouseDownPassword}
+                  edge="end"
+                >
+                  {isPasswordVisible ? <Visibility /> : <VisibilityOff />}
+                </IconButton>
+              </InputAdornment>
+            }
+            required
+          />
+        </FormControl>
+
+        {authError !== '' && (
+          <FormControl>
+            <Fade in={authError !== ''}>
+              <ErrorText>
+                <Translate>{authError}</Translate>
+              </ErrorText>
+            </Fade>
+          </FormControl>
+        )}
+
+        <FormControl>
+          {isFetching ? (
+            <CircularProgressCentered />
+          ) : (
+            <LoginButton type="submit" variant="contained" size="large">
```

Comment:
Perso je trouve pas le boutton de Log in très jolie :p 
Peut -etre rajouter les props 
`variant="text" color="inherit"`

![image](https://user-images.githubusercontent.com/7127156/82445757-b084dc80-9aa5-11ea-8a8d-589161b4bc49.png)

Reply:
J'ai hésité, va pour le "text & inherit" oui !